site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+                  <input class="form-check-input" type="checkbox" value="" id="fifthStorageFilter">
+                  <label class="form-check-label" for="fifthStorageFilter">
+                    1 TB (1)
+                  </label>
+                </div>
+              </div>
+            </div>
+
+            <div class="mb-2 pt-4">
+              <h3 class="h4 mb-0">SIM</h3>
+              <hr class="border-1 border-light my-2">
+              <div class="d-flex flex-column">
+                <div class="form-check mb-2">
+                  <input class="form-check-input" type="checkbox" value="" id="monoSimFilter">
+                  <label class="form-check-label" for="monoSimFilter">
+                    Mono (15)
+                  </label>
+                </div>
+                <div class="form-check mt-1 mb-2">
+                  <input class="form-check-input" type="checkbox" value="" id="dualSimFilter" disabled>
+                  <label class="form-check-label" for="dualSimFilter">
+                    Dual (0)
+                  </label>
+                </div>
+                <div class="form-check mt-1 mb-2">
+                  <input class="form-check-input" type="checkbox" value="" id="eSimFilter">
+                  <label class="form-check-label" for="eSimFilter">
+                    eSIM (3)
+                  </label>
+                </div>
+              </div>
+            </div>
+
+            <div class="mb-2 pt-4">
+              <h3 class="h4 mb-0">Price (FCFA)</h3>
+              <hr class="border-1 border-light my-2">
+              <div class="d-flex flex-column">
+                <div class="mt-1 mb-2">
+                  <label for="minusPriceFilter" class="form-label">From</label>
+                  <input type="email" class="form-control" id="minusPriceFilter" placeholder="Minimum price" value="50 000">
+                </div>
+                <div class="mt-1 mb-4">
+                  <label for="maxPriceFilter" class="form-label">To</label>
+                  <input type="email" class="form-control" id="maxPriceFilter" placeholder="Maximum price" value="300 000">
+                </div>
+                <div>
+                  <p class="placeholder-glow mb-0">
+                    <span class="visually-hidden">placeholder for double range slider</span>
+                    <span class="placeholder col-12" style="min-height: 50px;"></span>
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <hr class="d-lg-none border-1 border-light">
+
+            <button type="button" class="btn btn-primary d-lg-none mb-3">
+              See 23 results
+            </button>
+
+            <button type="button" class="btn btn-dark mb-3 order-lg-first">
+              <svg class="me-1" width="1.25rem" height="1.25rem" aria-hidden="true" focusable="false">
+                <use xlink:href="#delete"/>
+              </svg>
+              Reset filter
```

Comment:
Active filters: Purchase type, Brands, Colors and Storage.

So 'Reset filters'? Or at least 'Reset filter(s)'?

Reply:
I don't really what's best here, I think it depends on the way you see it. For me, the combination of the different option represent the filter so to my pov, it looks fine. We might ask designers about this.

Reply:
Waiting for designers, we can keep this comment open until we reach an agreement.